packages/next/client/image.tsx

```diff
@@ -209,6 +218,24 @@ function defaultImageLoader(loaderProps: ImageLoaderProps) {
   )
 }
 
+function removePlaceholder(
+  element: HTMLImageElement | null,
+  placeholder: PlaceholderValue
+) {
+  const hasBlurryPlaceholder =
+    configPlaceholder === PlaceholderValue.BLURRY &&
```

Comment:
Why does the configPlaceholder have to be blurry? If the configPlaceholder default is set to "none" but we manually override it with a `placeholder="blurry"` in the component attributes, don't we want that to take precedence?

Reply:
I think I misnamed the config value, leading to the confusion. I think eventually this should be the behavior, but right now the config value is simply used to turn the experimental feature on. I'll refactor to make this more clear.